x-pack/test/security_solution_cypress/cypress/e2e/investigations/alerts/detection_page_filters.cy.ts

```diff
@@ -220,35 +217,47 @@ describe.skip(`Detections : Page Filters`, { tags: ['@ess', '@serverless'] }, ()
     cy.get(FILTER_GROUP_CHANGED_BANNER).should('be.visible');
   });
 
-  context('with data modificiation', () => {
-    it(`Alert list is updated when the alerts are updated`, () => {
-      // mark status of one alert to be acknowledged
-      selectCountTable();
-      cy.get(ALERTS_COUNT)
-        .invoke('text')
-        .then((noOfAlerts) => {
-          const originalAlertCount = noOfAlerts.split(' ')[0];
-          markAcknowledgedFirstAlert();
-          waitForAlerts();
-          cy.get(OPTION_LIST_VALUES(0)).click();
-          cy.get(OPTION_SELECTABLE(0, 'acknowledged')).should('be.visible').trigger('click');
-          cy.get(ALERTS_COUNT)
-            .invoke('text')
-            .should((newAlertCount) => {
-              expect(newAlertCount.split(' ')[0]).eq(String(parseInt(originalAlertCount, 10)));
-            });
-        });
-    });
+  context('with data modification', () => {
+    /*
+     *
+     * default scrollBehavior is true, which scrolls the element into view automatically without any scroll Margin
```

Comment:
Thanks for the comment!

Reply:
As an aside, I feel like the popover should re-position based on it's position in the container...maybe a suggestion for Eui..

Reply:
it is not a EUI popover, but a custom `onHover` element. See screenshot below:

<img width="717" alt="Screenshot 2024-04-24 at 18 17 48" src="https://github.com/elastic/kibana/assets/7485038/4cc74e1b-ebe4-4a4a-9856-78969700d645">